src/integrationTest/java/uk/gov/hmcts/reform/orgrolemapping/scheduler/PrmSchedulerProcess4IntegrationTest.java

```diff
@@ -0,0 +1,491 @@
+            NEW_USER_LAST_UPDATED, true, false);
+    }
+
+    /**
+     * Retry - Failed.
+     */
+    @Test
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = {
+        "classpath:sql/prm/organisation_refresh_queue/init_organisation_refresh_queue.sql",
+        "classpath:sql/prm/organisation_refresh_queue/insert_organisation4.sql",
+        "classpath:sql/prm/user_refresh_queue/init_user_refresh_queue.sql"
+    })
+    void testRetryFailed() {
+
+        // verify that the organisations are attempted to be updated 3 times
+        runTest(List.of("/SchedulerTests/PrdUsersByOrganisation/userOrganisation4_scenario_01.json"),
+            EndStatus.FAILED, 3);
+
+        // verify that the OrganisationRefreshQueue contains 1 record, 1 active, 4 retries
+        assertTotalOrganisationRefreshQueueEntitiesInDb(1, 1, 4);
+
+        // Verify no active users in the refresh queue
+        assertTotalUserRefreshQueueEntitiesInDb(0);
+    }
+
+    private void runTest(List<String> fileNames, EndStatus endStatus, int noOfCallsToPrd) {
+
+        // GIVEN
+        logBeforeStatus();
+        stubPrdRetrieveUsersByOrg(fileNames, "false", null, endStatus);
+
+        // WHEN
+        ProcessMonitorDto processMonitorDto = prmScheduler
+            .findUsersWithStaleOrganisationsAndInsertIntoRefreshQueueProcess();
+
+        // THEN
+        verifyNoOfCallsToPrd(noOfCallsToPrd, endStatus);
+        logAfterStatus(processMonitorDto);
+
+        // verify that the process monitor reports the correct status
+        assertEquals(endStatus, processMonitorDto.getEndStatus());
+    }
+
+    //#region Assertion Helpers: DB Checks
+
+    private void assertTotalOrganisationRefreshQueueEntitiesInDb(int expectedNumberOfRecords, int expectedActiveOrgs,
+        int expectedRetries) {
+        var organisationRefreshQueueEntities = organisationRefreshQueueRepository.findAll();
+        assertEquals(expectedNumberOfRecords, organisationRefreshQueueEntities.size(),
+            "OrganisationRefreshQueueEntity number of records mismatch");
+        int activeOrgs = 0;
+        int retries = 0;
+        for (var entity : organisationRefreshQueueEntities) {
+            if (entity.getActive()) {
+                activeOrgs++;
+            }
+            retries += entity.getRetry() != null ? entity.getRetry() : 0;
+            assertRetryAfter(entity.getOrganisationId(), entity.getRetry(), entity.getRetryAfter());
+        }
+        assertEquals(expectedActiveOrgs, activeOrgs,
+            "OrganisationRefreshQueueEntity active organisations count mismatch");
+        assertEquals(expectedRetries, retries,
+            "OrganisationRefreshQueueEntity retries count mismatch");
+    }
+
+    private void assertRetryAfter(String organisationId, Integer retry, LocalDateTime retryAfter) {
+        // Successful retry or retry limit exceeded
+        if (retry == 0 || retry >= 3) {
```

Comment:
to align with the business requirement, please change this condition to : retry >= 4